html/moon_bootz/legal/waiver.html

```diff
@@ -6,8 +6,19 @@
 <body>
   <p>Moon Bootz (TM) is not responsible for anything that happens to you while wearing their product. Ever. You might want to invest in a helmet.</p>
 
-<footer>
-       <a href="file:///Users/MarenMcMullan/devbootcamp/phase-0-tracks/html/moon_bootz/index.html">Back to home</a>
-</footer>
+  <p><img src="../images/high_jump.jpg" alt="high_jump" /></p>
+
+  <p>By Richard Kelland</p>
```

Comment:
Hey @Marenmc This all looks good but are we sure that Richard Kelland is the original author of this?